packages/vertica-nodejs/lib/client.js

```diff
@@ -517,7 +528,9 @@ class Client extends EventEmitter {
       client_os: params.client_os,
       client_os_user_name: params.client_os_user_name,
       client_os_hostname: params.client_os_hostname,
-      client_pid: params.client_pid
+      client_pid: params.client_pid,
+      binary_data_protocol: '0',  // Defaults to text format '0'
+      protocol_compat: 'VER',
```

Comment:
Since OAuth in vertica-nodejs was not supported when those old servers were released we can just say this is a new feature that works with newer server versions starting at the version where we removed access token from the startup message.